app/misc/storeCredentials.ts

```diff
@@ -31,50 +18,25 @@ function encryptAccessToken(accessToken) {
   })
 }
 
+// Returns the un-encrypted access token that is stored in memory
 function getAccessToken(){
-  if (encryptAccessToken === undefined)
-    return null;
-  else {
-    var decryptedAccessTokenBytes = CryptoJS.AES.decrypt(encryptedUsername.toString(), os.hostname());
-    return decryptedAccessTokenBytes.toString(CryptoJS.enc.Utf8);
-  }
+  // If user is not logged in, return null
+  if (!!!encryptedToken)
+    return null
+  // Decrypt the access token to 
+  var bytes = CryptoJS.AES.decrypt(encryptedToken.toString(), os.hostname());
+  return bytes.toString(CryptoJS.enc.Utf8);
 }
 
-function encryptTemp(username, password) {
-  encryptedUsername = CryptoJS.AES.encrypt(username, os.hostname());
-  encryptedPassword = CryptoJS.AES.encrypt(password, os.hostname());  
-}
-
-function getUsernameTemp() {
-  if (encryptedUsername === undefined){ // the user has not logged in, return null
-    return null;
-  }else {
-    var decryptedUsernameBytes = CryptoJS.AES.decrypt(encryptedUsername.toString(), os.hostname());
-    return decryptedUsernameBytes.toString(CryptoJS.enc.Utf8);
-  }
+// Takes in the un-encrypted access token, then encrypts it and stores it in memory
+function encryptTemp(token) {
+  encryptedToken = CryptoJS.AES.encrypt(token, os.hostname());
 }
 
+// Returns the password used for Oauth Credentials
 function getPasswordTemp() {
```

Comment:
since this is only called from authenticate.ts, I do not see the value of it being in this file -- especially since it returns a hard coded string. Any reason not to move this as a constant to authenticate.ts?

Reply:
Once again, since the password is used amongst multiple files, I am hesitant to reference a const directly which has been declared in another file but I understand your point.